app/src/main/java/com/teamagam/gimelgimel/app/common/utils/UsernameGenerator.java

```diff
@@ -0,0 +1,35 @@
+package com.teamagam.gimelgimel.app.common.utils;
+
+import android.content.Context;
+import android.content.res.Resources;
+
+import com.teamagam.gimelgimel.R;
+
+import java.util.Random;
+
+public class UsernameGenerator {
+
+    private static final String SPACE = " ";
+    private static final Random rand = new Random();
+
+    public static String generateUsername(Context context) {
+        Resources res = context.getResources();
+        String[] animals = res.getStringArray(R.array.name_generator_animals);
+        String[] adjectives = res.getStringArray(R.array.name_generator_adjectives);
+        return capitalizeFirstLetters(adjectives[rand.nextInt(adjectives.length)]) + SPACE +
```

Comment:
I'd also use a method named pickRandom(array) that picks at random a member of an array and use it